database/models/events.db.php

```diff
@@ -263,7 +267,7 @@ static function fromEventList(array $events)
                 $event['end_date'],
                 $event['deadline'],
                 $event['open'],
-                isset($event['present']) ? $event['present'] : null
+                isset ($event['present']) ? $event['present'] : null
```

Comment:
This change is a bit odd. I am not familiar enough with PHP logic to know if it breaks anything

Reply:
See above 👇